sample/index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
```

Comment:
No images, no styling, just minimal HTML and perhaps `max-width: 800px; margin: 0 auto` for the body, so the editor doesn't take the whole width.